commands/root.go

```diff
@@ -84,10 +84,6 @@ func NewRootCmd(name string, isPlugin bool, dockerCli *command.DockerCli) *cobra
 		"using default config store",
 	))
 
-	if !confutil.IsExperimental() {
```

Comment:
Also considering if we should just (as we did for the CLI) always enable the `debug` command;

https://github.com/docker/buildx/blob/08dd378b5943ef48a4258ca7c640c309d1a7fdb9/commands/root.go#L121-L123


The output already shows it's experimental;

```console
BUILDX_EXPERIMENTAL=1 docker buildx debug --help
Usage:  docker buildx debug [OPTIONS] COMMAND

Start debugger (EXPERIMENTAL)

EXPERIMENTAL:
  docker buildx debug is an experimental feature.
  Experimental features provide early access to product functionality. These
  features may change between releases without warning, or can be removed from a
  future release. Learn more about experimental features in our documentation:
  https://docs.docker.com/go/experimental/
```

Reply:
In the future I agree that we should deprecate `BUILDX_EXPERIMENTAL` as the CLI already marks commands as experimental anyway.

Reply:
For what it's worth, the debug adapter stuff is going under another command in `dap`. Right now, `debug` controls the monitor. We do potentially have plans to completely remove the `debug` command and integrate it directly with build.

Reply:
Discussed with @tonistiigi, we would need to always enable the `debug` command if we are removing this hint from help so users are aware of it.